### DISCUSSION TITLE
Unable to visualize Diagnostics

### DISCUSSION THREAD

**Discussion by jgruiz75** [microsoft/psi#221] [Q&A] [answered]
I have enabled Diagnostics and can see that it is being written to the store (MessageCount also shows messages have been received). However, whenever I try to use the graph visualizer I do not see anything, and moving over the timeline prompts an "Instant Data Push Error" popup message.

Any ideas? Or any ideas on how to debug this issue?

**Comment by sandrist** [comment, marked as answer]
Can you try running PsiStudio from Visual Studio with a debugger attached? It will be informative to catch exactly where in the code this exception is being thrown from.

**Reply by jgruiz75** [reply]
You caught me Sean, I was using an old binary of PSI studio. Recompiling the newest version fixed the issue.

If anyone finds this post, make sure you are using the PSI studio corresponding to the PSI framework version you are using.

**Reply by sandrist** [reply]
Glad you got it working!
